src/useVH.ts

```diff
@@ -1,39 +1,36 @@
-import { useCallback, useLayoutEffect, useState } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 
 const VARIABLE_NAME = '--vh';
 
-const getActualVh = () =>
-  // window check for server-side rendering
-  typeof window !== 'undefined'
-    ? Number((window.innerHeight * 0.01).toFixed(2))
-    : 0;
+const getActualVh = () => Number((window.innerHeight * 0.01).toFixed(2));
 
 let count = 0;
 
-const useVH = (): number => {
-  const [vh, setVh] = useState(getActualVh());
+export const useVH = (): number => {
+  const [vh, setVh] = useState(0);
   const updateVh = useCallback(() => {
     const newVh = getActualVh();
 
     document.documentElement.style.setProperty(VARIABLE_NAME, `${newVh}px`);
     setVh(newVh);
   }, [setVh]);
 
-  useLayoutEffect(() => {
-    count += 1;
+  useEffect(() => {
+    if (typeof window !== 'undefined') {
```

Comment:
instead of this better to use an early return

Reply:
I think the current linter does not allow an early return because of the useEffect unmount return